### DISCUSSION TITLE
Packet isn't being sent using Multicast

### DISCUSSION THREAD

**Discussion by rs0909** [ANLAB-KAIST/KENSv3#246] [Project Q&A 3. Internet Layer (IP/Routing)] [answered]
We've been trying to send packet with src_ip as getipaddr(0), and dest_ip as multicast (224.0.0.5). But router_id in the header of arrived packet, router_id is same with getIpAddr(0). And nothing is being recorded in Pcap file too. 

I think the packet is not being sent to other routers but it is just coming back to the sending router. What are the cases that we implemented wrong??

**Comment by Pusnow** [comment, marked as answer]
Would you send your source code to us? cs341.pj@kaist.ac.kr

**Comment by Pusnow** [comment]
You should not use `htonl` after `arrayToUINT64`.

**Reply by rs0909** [reply]
We are just trying to send packet with only source Ip and multicast ip and still the packet isn't sent and nothing is shown in cap file.

**Reply by rs0909** [reply]
The problem was caused by as we didn't clone every file but only PWOSPF folder. Cloned all over again and it works!
